domains/faultynet.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "FaultyWindows",
+           "email": "faultywindows@gmail.com",
+           "discord": "1041028534690386051"
+        },
+    
+        "record": {
+            "CNAME": "faultywindows.github.io"
```

Comment:
Please add more stuff to your [website](https://faultywindows.github.io/FaultyNet/)

Reply:
I've made it very simple only because I didn't work on the website beforehand, but now I have, so i'll import those files.